x-pack/packages/ml/aiops_utils/src/fetch_stream.ts

```diff
@@ -54,6 +56,7 @@ export async function* fetchStream<I extends UseFetchStreamParamsDefault, BasePa
         // This refers to the format of the request body,
         // not the response, which will be a uint8array Buffer.
         'Content-Type': 'application/json',
+        'Elastic-Api-Version': apiVersion,
```

Comment:
There is a constant for this header label shared from core.

`import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';`

Reply:
Thanks! Fixed in 8660318.